### ISSUE TITLE
Image distortion in PDF

### ISSUE THREAD

**Issue by stefan-plate** [apprenticeharper/DeDRM_tools#1056]
I've had this problem for a while, now checked again using the latest Calibre Plugin (6.7) on macOS 10.15:
Some images in the decrypted PDF are distorted like this:
<img width="230" alt="Screen Shot 2020-04-11 at 17 09 23" src="https://user-images.githubusercontent.com/57003499/79102228-99e2bb80-7d6a-11ea-8ef2-86968b742b66.png">
While the original looks like this:
<img width="125" alt="Screen Shot 2020-04-11 at 17 13 52" src="https://user-images.githubusercontent.com/57003499/79102257-a8c96e00-7d6a-11ea-9d56-f82718633eca.png">

**Comment by ElleKayEm** [comment]
Is this in the calibre viewer?  Or the same program that you're using to view the original pdf?

**Comment by stefan-plate** [comment]
It looks excactly the same in macOS Preview and Adobe Acrobat.

**Comment by ElleKayEm** [comment]
No conversion?  Just DRM removal?

**Comment by stefan-plate** [comment]
Absolutely. Downloaded with Adobe Digital Editions (current version), pulled the PDF into calibre (with current DeDRM Plugin) and opened the resulting file. Just DRM removal, no conversion. BTW the result is the same if I use the standalone DeDRM app (which I understand is not updated anymore).

**Comment by ElleKayEm** [comment]
Seems like something @apprenticeharper will need to look at.  It may be a while before he gets a chance to see this here.  You could try contacting him at apprenticeharper@gmail.com.

**Comment by rosshjb** [comment]
From https://www.adobe.com/kr/solutions/ebook/digital-editions/faq.html :

> **Why do some PDF documents look different in Digital Editions than they do in Acrobat and Reader?**
>
> Digital Editions utilizes Adobe PDF technology that has been optimized for small code size and lower system requirements. It is optimized for performance and onscreen readability, rather than graphic arts or prepress-level fidelity. As a result, some minor differences in graphics and font rendering are to be expected. In addition, the Digital Editions PDF implementation does not support several enterprise-oriented PDF features. These features are not typically used in PDF-based eBooks, and if present, they are ignored but may cause certain documents to render incorrectly.
